src/mailer/mailerTemplate.ts

```diff
@@ -0,0 +1,4 @@
+export const MailerTemplate: { [key: string]: any } = {
```

Comment:
Let’s call this `MailTemplate` instead and make it a real `enum`. As noted we should move to having this be the interface for both html/text template paths but instead with simple keys:
```suggestion
export enum MailTemplate {
  contactUs = 'contactUs',
}

export const templates = {
  html: {
    [MailTemplate.contactUs]: 'src/mailer/views/html/contactUs.html',
  },
  text: {
    [MailTemplate.contactUs]: 'src/mailer/views/text/contactUs.txt',
  },
}
```


This can also be exported from within `src/mailer/index.ts` as a shortcut.
```ts
export * from ‘./mailTemplate’
```

Reply:
Complete